### DISCUSSION TITLE
[A-L] (16 Gennaio 2023) Esercizio 4

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#367] [Esami A-L]
Sia $\Large R$ la relazione seguente $\Large \\{ (1, 2),\ (1, 3),\ (2, 4),\ (3, 4),\ (4, 5)\\}$ sull’insieme $\Large \\{1, 2, 3, 4, 5\\}$.

1. R è una relazione di ordine totale su $\Large \\{1, 2, 3, 4, 5\\}$?

2. Calcolare $\Large R \circ R$.

3. Calcolare la chiusura transitiva di $\Large R$.

**Comment by cek753** [comment]
1- Falso poiché non rispetta riflessività, transitività e totalità
2- (1,4), (2,5), (3,5)
3- (1,4), (1,5), (2,5), (3,5)
